### PR TITLE
Introduce composer docs:generate

### PR DESCRIPTION
**Pull request by linawolf**
A command to locally generate the documentation. Idea taken from EXT:tea

This is a first proof of concept. I think this would make rendering locally even easier for people. What do you think? @marble @brotkrueml ?

### CODE REVIEW

**Review by brotkrueml — COMMENTED** [review]
I am not a fan of custom composer scripts which can also be added simply via a shell script (which you obviously do in the composer command; just run a sequence of shell commands).
So, I would add a shell script (also keep Windows in mind, I assume bash is there not available by default - apart from WSL2 - independently, if composer or not).

Don't know if this has a benefit, I see no issue in running two commands in sequence initially and then just run the second command only afterwards - as I recall the sh history quickly.

So, would leave the decision to others.

**Comment by linawolf** [comment]
I would see this as an offer, none would force anyone to do it this way. I get questions from confused people all over again: How can I render the documentation? This way it would kind of documented in place. Just calling "composer" even lists you the script in the help section:
image

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+composer docs:generate
+```
+
+
 ## Create Issues
 
-* If you find something missing or something is wrong in this manual, you are welcome to write an issue describing the problem. 
-* If you can, please try to fix the problem yourself. 
-* For minor changes, it is not necessary to create an issue first. 
+* If you find something missing or something is wrong in this manual, you are welcome to write an issue describing the problem.
```
Comment (mabolek) [inline CONTRIBUTING.md:15]:
```suggestion
* If you find something missing or something is wrong in this manual, you are welcome to open an issue and describe the problem.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+* If you can, please try to fix the problem yourself.
```
Comment (mabolek) [inline CONTRIBUTING.md:16]:
```suggestion
* Please feel free to submit a fix to the problem.
```

**Review by mabolek — CHANGES_REQUESTED** [review]

**Comment by linawolf** [comment]
I think Gitpod is the better solution

**Pull request closed without merging**